### PR TITLE
Add option to use PadeDelay instead of normal delay to be used for linearisation

### LINKED ISSUE #277: GGOV1 variant for linearization or update using Pade model
@ManuelNvro and @dietmarw 

@sergio-dorado reported an issue over email that I will add below.

When attempting to linearize certain blocks (e.g. the delay=`Modelica.Blocks.Nonlinear.FixedDelay` model), any Modelica tool should fail.  This is expected when using that model as it is not possible to linearize it, it does not use either functions for derivatives or a smooth operator-based implementation.

Most user's of this library will not understand this, and think there is a problem with the tool or the library.

We have two choices, 

1. either create model variants that are "fit for linearization" that can be easily replaced, 
2. or that we replace all "delay" blocks with a Pade approximation from (`Modelica.Blocks.Nonlinar.PadeDelay`) 

@marcelofcastro if we replace with the Pade approximation we have to run all the CI tests to made sure we have the new results for all models that have the change...

My preference would be for no. 2, because I don't think the user's will pay attention to the availability of a variant, but that of course gives more work in running CI tests.
What do you think @marcelofcastro and @dietmarw?

Issue report from @sergio-dorado (please do this using the Issues tab next time!):

> ---------- Forwarded message ----------
> From: Sergio A. Dorado-Rojas <sergio.dorado.rojas@gmail.com>
> Date: Jul 31, 2021, 7:53 AM -0400
> To: Marcelo de Castro Fernandes <decasm3@rpi.edu>, Giuseppe Laera <laerag@rpi.edu>
> Cc: Luigi Vanfretti <luigi.vanfretti@gmail.com>
> Subject: Issues with Delay Block for Linearization
> 
> Hi all,
> 
> I have been working with an OpenIPSL model for my internship. I was trying to perform a linear analysis to change some AVR’s parameters and found something that I think is worth to discuss.  The model has a single generator with TG (GGOV1) and ES (ESST4B). Marcelo did the translation for me so it is working on the latest OpenIPSL version. I did not create this as an issue on GitHub because ... **redacted.**

Attachment-2
> 
> Whenever I try to use any function within the Linear Analysis menu, I get this message. The linear analysis is not carried out.
> 
Attachment
> 
> 
> So, I traced down the model for delay blocks and found that there is one in the turbine model. 
> 
Attachment-2

> 
> 
> For my particular case, the delay time was exactly zero so I removed the block. The linear analysis functions could be used perfectly afterwards.
> 
> In my humble opinion, it doesn’t speak highly for the library that the linear functions cannot be used for such blocks. I’d suggest to replace the linear block by a Padé approximation. However, I don’t know much of how this would impact the exactitude of the model against its PSS/E counterpart. I haven’t tried this workaround myself since I am not familiar at all with this particular exciter’s structure and, as I mentioned above, for my immediate needs it was better to remove the block. 
> 
> Let me know what your thoughts are. 
> 
> Best,
> Sergio

### PR DESCRIPTION
**Pull request by dietmarw**
Fixes #277

### CODE REVIEW

**Comment by lvanfretti** [comment]
@dietmarw there where some problems when attempting to simulate using the Pade model, so I made so changes to your suggestion I made in a PR on your fork at https://github.com/dietmarw/OpenIPSL/pull/1

**Comment by dietmarw** [comment]
@lvanfretti Thanks for the extension. That works even nicer. I've made some small extensions/corrections. You could also have directly pushed into the PR here instead of creating a PR on this PR. But that worked of course also .

File: OpenIPSL/Tests/Controls/PSSE/TG/package.order [hunk 1]
```diff
@@ -1,5 +1,6 @@
 GAST
 GGOV1
+GGOV1Pade
```
Comment (lvanfretti) [inline OpenIPSL/Tests/Controls/PSSE/TG/package.order:3]: Need to remove line 3, that was my mistake, I was planning to make a test model for the Pade variant, however, not possible at the time being.

**Review by lvanfretti — CHANGES_REQUESTED** [review]
Remove line 3 in OpenIPSL/Tests/Controls/PSSE/TG/package.order, model does not exist. Added by @lvanfretti by mistake to package.order after having removed it...

File: OpenIPSL/Tests/Controls/PSSE/TG/package.order (hunk 1 above)
Comment (dietmarw) [inline OpenIPSL/Tests/Controls/PSSE/TG/package.order:3]:
```suggestion
```

Comment (dietmarw) [reply OpenIPSL/Tests/Controls/PSSE/TG/package.order]: removed.

**Review by lvanfretti — COMMENTED** [review]
Thanks for fixing my bug!

**Pull request merged**